examples/README.md

```diff
@@ -1,48 +0,0 @@
-## Examples

```

Comment:
Please keep "root" README.md with description for existing examples. Something like:

```
## Examples

### Basic

- [IOxExample](examples/src/main.ts) - How to use write and query data from InfluxDB IOx

### Web env

- [Browser](examples/src/main.ts) - How to use client in web env

```

will be pretty useful.